pkg/proxy/apis/config/types.go

```diff
@@ -138,9 +138,6 @@ type KubeProxyConfiguration struct {
 	// portRange is the range of host ports (beginPort-endPort, inclusive) that may be consumed
 	// in order to proxy service traffic. If unspecified (0-0) then ports will be randomly chosen.
 	PortRange string
-	// resourceContainer is the absolute name of the resource-only container to create and run
-	// the Kube-proxy in (Default: /kube-proxy).
-	ResourceContainer string
```

Comment:
You must also remove the default value for `ResourceContainer` (https://github.com/kubernetes/kubernetes/blob/master/pkg/proxy/apis/config/v1alpha1/defaults.go#L61-L63)

Reply:
Thanks, I spotted that but I'm still missing something. All unit tests are fine, but the system doesn't start properly in e2e.